PlayFabSDK/source/Qos/RegionPinger.cs

```diff
@@ -122,5 +179,24 @@ private bool VerifyResponseBuffer(byte[] header, long ticks)
             return BitConverter.ToInt64(header, _subsequentHeader.Length) == ticks;
```

Comment:
_subsequentHeader is always a byte[] of {0x00, 0x00};
It's never changed.
This whole function seems very strange to me.
It's fine to compare the input/output ticks, but I don't see how _subsequentHeader plays any role in that.

Reply:
It is a sanity check to make sure we are checking the correct packet. For example, if we somehow got the ping packet instead of the response packet the ticks check would pass, but the subsequent header check would fail.  Obviously not required, but also should be an (extremely) inexpensive check.

---
In reply to: [376149481](https://github.com/PlayFab/CSharpSDK/pull/111#discussion_r376149481) [](ancestors = 376149481)

Reply:
So what I think you're saying is that the ping packets have an expected fixed-header*, which is zero-zero.
* Is header the right word? Maybe prefix? It's just the first two bytes of the body, not an actual header...